starboard/nplb/media_can_play_mime_and_key_system_test.cc

```diff
@@ -53,6 +53,26 @@ bool IsKeySystemWithAttributesSupported() {
   return false;
 }
 
+typedef enum DeviceType {
+  // These enums must not be reordered.
```

Comment:
Maybe expand this comment to say something like "enums must not be reordered to ensure all test cases are run correctly below"?

Alternatively you could also use a switch below which would remove this need for no-reordering e.g.

```
DeviceType device_type = GetDeviceType();
switch(device_type) {
  case kDeviceType8k:
      assert_mime_support(params_8k);
      [[fallthrough]];
  case kDeviceType4k:
     assert_mime_support(params_4k);
      [[fallthrough]];
  default:
    assert_mime_support(params_1080p);
}
```

Reply:
I like the switch option, made the change.

Reply:
you should add the `[[fallthrough]]` attribute - it was added in c++17 so should be supported, and indicates to the reader that fallthrough is intended. https://en.cppreference.com/w/cpp/language/attributes/fallthrough